nipoppy_cli/nipoppy/data/examples/sample_global_config-all_pipelines.json

```diff
@@ -0,0 +1,161 @@
+{
+    "DATASET_NAME": "<DATASET_NAME>",
+    "VISITS": [
+        "<VISIT_LABEL>"
+    ],
+    "SESSIONS": [
+        "ses-<SESSION_LABEL>"
+    ],
+    "CONTAINER_CONFIG": {
+        "ARGS": [
+            "--cleanenv"
+        ]
+    },
+    "GLOBALS": {
+        "[[HEUDICONV_HEURISTIC_FILE]]": "<PATH_TO_HEURISTIC_FILE>",
```

Comment:
Why is it here and also in `BIDS_PIPELINES` :-/

Reply:
This is the template/placeholder string that will be replaced by whatever the user puts on the right side!